tasks/dart/easy/add_numbers.dart

```diff
@@ -6,3 +6,6 @@ void main() {
 }
 
 // TODO: Implement addNumbers function
+int addNumbers(int1, int2) {
```

Comment:
Code looks mostly good! However this does not quite work, since the types of the parameters needs to be specified. So it should be
```dart
int addNumbers(int int1, int int2) {
   return int1 + int2;
}
```

Im still going to merge it, no need to fix something that small. Remember always to test your code before submitting it to prevent errors.